### PR TITLE
Migrates buildpack in paketo-buildpacks org

### PR DESCRIPTION
**Pull request by ForestEckhardt**
## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by ForestEckhardt** [comment]
@paketo-buildpacks/utilities-maintainers Could I get an approval from someone on this?

**Review by dmikusa-pivotal — DISMISSED** [review]

File: buildpack.toml
Comment (dmikusa-pivotal) [inline buildpack.toml:5]:
```suggestion
name = "Paketo Source Removal Buildpack"
```

**Review by dmikusa-pivotal — APPROVED** [review]

**Pull request merged**
